### PR TITLE
add rules for volume interaction via IOCTLs

### PR DESCRIPTION
**Pull request by williballenthin**
from https://www.crowdstrike.com/blog/the-anatomy-of-wiper-malware-part-3/

### CODE REVIEW

File: host-interaction/driver/interact-with-driver-via-control-codes.yml
Comment (mr-tz) [inline host-interaction/driver/interact-with-driver-via-control-codes.yml:1]: should we standardize this to also use `via ioctl`?

**Comment by williballenthin** [comment]
yeah let me refactor a few things here. i'll make `interact with driver via ioctl` the main place that we reference `DeviceIoControl` and move the driver load/unload to other rules.

File: nursery/set-system-properties.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+rule:
+  meta:
+    name: set system properties
```
Comment (mr-tz) [inline nursery/set-system-properties.yml:3]: this loads a file path / driver, see https://is.muni.cz/th/y8opj/Bc.pdf
rename the rule and file?

**Review by mr-tz — APPROVED** [review]
LGTM besides the name of one rule

File: nursery/set-system-properties.yml (hunk 1 above)
Comment (williballenthin) [reply nursery/set-system-properties.yml:3]: ah, ok, i'll move that over to `install driver`

**Pull request merged**